pymc3/tests/test_distributions_random.py

```diff
@@ -1348,6 +1344,35 @@ def dist(cls, **kwargs):
                 pymc3_random(TestedInterpolated, {}, ref_rand=ref_rand)
 
 
+class TestKroneckerNormal(BaseTestDistribution):
+    def kronecker_rng_fn(self, size, mu, covs=None, sigma=None, rng=None):
+        cov = pm.math.kronecker(covs[0], covs[1]).eval()
+        cov += sigma ** 2 * np.identity(cov.shape[0])
+        return st.multivariate_normal.rvs(mean=mu, cov=cov, size=size)
+
+    pymc_dist = pm.KroneckerNormal
+
+    n = 3
+    N = n ** 2
+    covs = [RandomPdMatrix(n), RandomPdMatrix(n)]
+    mu = np.random.random(N) * 0.1
+    sigma = 1
+
+    pymc_dist_params = {"mu": mu, "covs": covs, "sigma": sigma}
+    expected_rv_op_params = {"mu": mu, "covs": covs, "sigma": sigma}
+    reference_dist_params = {"mu": mu, "covs": covs, "sigma": sigma}
+    sizes_to_check = [None, (), 1, (1,), 5, (4, 5), (2, 4, 2)]
+    sizes_expected = [(N,), (N,), (1, N), (1, N), (5, N), (4, 5, N), (2, 4, 2, N)]
+
+    reference_dist = lambda self: functools.partial(
+        self.kronecker_rng_fn, rng=self.get_random_state()
+    )
+    tests_to_run = [
+        "check_pymc_draws_match_reference",
+        "check_rv_size",
```

Comment:
```suggestion
        "check_pymc_params_match_rv_op",
        "check_pymc_draws_match_reference",
        "check_rv_size",
```

Any reason not to test the `params_match`?

Reply:
Oh, that fails because the way we're handling the `cov` parameter internally. (as `*cov`) . So to that test it appears as a list of `TensorVariable` instead of a single `TensorVariable`

Reply:
I see. Do you think there is any point in implementing an alternative test for this logic or given things are working on the logp / random side, it should be fine?

Reply:
It should be fine, I think.